rfcs/0002-principles.md

```diff
@@ -0,0 +1,61 @@
+---
+created-on: 2021-04-17
+freshness: 2021-04-17
+owner: @dj2
+status: proposal
+---
+
+[La version française suit.](#résumé)
+
+# Principles
+
+## Summary
+This RFC lays out the guiding principles for the developer community at CDS.
+These principles guide how we work, what we work on and why we choose projects.
+
+
+## Motivation
+In order to better work with each other and the rest of CDS it's good to lay out
+the guiding principles of the community. These principles serve as guide posts
+for the current community but also serve as a foundation for new CDS employees
+to understand how we work.
+
+
+## Proposal
```

Comment:
Principles help translating our values into action. I feel we could map our principles with the CDS values? https://docs.google.com/document/d/13mFNfvn1F-HaYBhSEENPdzxrn0Rtg_hFguECUB7CTfk/edit

Reply:
This is where I think we need to figure out part of the line between RFC and Discussion. Does it make sense for the RFC to lay out just the principles and then have a discussion topic for how those map to CDS Values? Or should all that go in the RFC? I'm not really sure of the best line at this point.

Reply:
That said, seeing your comments below, I think it could fit in here well. Both of those things should be slow changing so leaning towards the RFC being the place for the mapping in this instance.

Is there a public listing of the contents of that document somewhere on the site?

Reply:
sounds good, let me ask if we got a public version of our values

Reply:
If we don't, can we make it public? I can move this to 0003, and inject a 0002-values document in first, heh.

Reply:
Could we add something that indicates how important **data** privacy, integrity and security are for us?

Reply:
What about adding an entry under `Optimize for outcomes` saying something along the lines of:

 - Maintain the trust of our users by sufficiently protecting the privacy, security and integrity of their data


It's not great, still trying to find the right words, suggestions?

Reply:
done